mat64/dense_test.go

```diff
@@ -307,6 +307,79 @@ func (s *S) TestRowColView(c *check.C) {
 	}
 }
 
+func (s *S) TestRowColViewFullCap(c *check.C) {
+	for _, test := range []struct {
+		mat [][]float64
+	}{
+		{
+			mat: [][]float64{
+				{1, 2, 3, 4, 5},
+				{6, 7, 8, 9, 10},
+				{11, 12, 13, 14, 15},
+				{16, 17, 18, 19, 20},
+				{21, 22, 23, 24, 25},
+			},
+		},
+		{
+			mat: [][]float64{
+				{1, 2, 3, 4},
+				{6, 7, 8, 9},
+				{11, 12, 13, 14},
+				{16, 17, 18, 19},
+				{21, 22, 23, 24},
+			},
+		},
+		{
+			mat: [][]float64{
+				{1, 2, 3, 4, 5},
+				{6, 7, 8, 9, 10},
+				{11, 12, 13, 14, 15},
+				{16, 17, 18, 19, 20},
+			},
+		},
+	} {
+		rows, cols, flattened := flatten(test.mat)
```

Comment:
```
rows, cols, flat := flatten(test.mat)
m := NewDense(rows, cols, flat[:len(flat):len(flat)])
```

I think I'll also change flatten since that would have found this earlier. I'll send that.